scripts/audit_chains/audit_contracts_setup.js

```diff
@@ -0,0 +1,385 @@
+
+    log += ", address: " + veOLAS.address;
+    // Check current token
+    const token = await veOLAS.token();
+    customExpect(token, globalsInstance["olasAddress"], log + ", function: token()");
+
+}
+
+// Check buOLAS: chain Id, provider, parsed globals, configuration contracts, contract name
+async function checkBUOLAS(chainId, provider, globalsInstance, configContracts, contractName, log) {
+    // Check the bytecode
+    await checkBytecode(provider, configContracts, contractName, log);
+
+    // Get the contract instance
+    const buOLAS = await findContractInstance(provider, configContracts, contractName);
+
+    log += ", address: " + buOLAS.address;
+    // Check current token
+    const token = await buOLAS.token();
+    customExpect(token, globalsInstance["olasAddress"], log + ", function: token()");
+
+    // Check owner
+    const owner = await buOLAS.owner();
+    customExpect(owner, globalsInstance["timelockAddress"], log + ", function: owner()");
+}
+
+// Check wveOLAS: chain Id, provider, parsed globals, configuration contracts, contract name
+async function checkWrappedVEOLAS(chainId, provider, globalsInstance, configContracts, contractName, log) {
+    // Check the bytecode
+    await checkBytecode(provider, configContracts, contractName, log);
+
+    // Get the contract instance
+    const wveOLAS = await findContractInstance(provider, configContracts, contractName);
+
+    log += ", address: " + wveOLAS.address;
+    // Check current token
+    const token = await wveOLAS.token();
+    customExpect(token, globalsInstance["olasAddress"], log + ", function: token()");
+    
+    // Check ve
+    const ve = await wveOLAS.ve();
+    customExpect(ve, globalsInstance["veOLASAddress"], log + ", function: ve()");
+}
+
+// Check GolvernorOLAS: chain Id, provider, parsed globals, configuration contracts, contract name
+async function checkGovernorOLAS(chainId, provider, globalsInstance, configContracts, contractName, log) {
+    // Check the bytecode
+    await checkBytecode(provider, configContracts, contractName, log);
+
+    // Get the contract instance
+    const governor = await findContractInstance(provider, configContracts, contractName);
+
+    log += ", address: " + governor.address;
+    // Check current token
+    const token = await governor.token();
+    customExpect(token, globalsInstance["wveOLASAddress"], log + ", function: token()");
+
+    // Check timelock
+    const timelock = await governor.timelock();
+    customExpect(timelock, globalsInstance["timelockAddress"], log + ", function: timelock()");
+
+    // Check version, hardcoded
+    const version = await governor.version();
+    customExpect(version, "1", log + ", function: version()");
+
+    // Check quorumNumerator, hardcoded
+    const quorumNumerator = await governor["quorumNumerator()"]();
+    customExpect(quorumNumerator.toString(), globalsInstance["quorum"], log + ", function: quorumNumerator()");
```

Comment:
I would have expected that this and the the following two would have shown an issue. Indeed governorOLAS on goerli have different `quorum`, `initialVotingDelay`,  `initialVotingPeriod`. I guess that hardcoded values should be used here

Reply:
We compare values with our globals files - they are different for mainnet and goerli. If we want this to be exactly as on the mainnet, we need to use its values then.

Reply:
Updated with the mainnet values.